locale/ru_RU/countries.xml

```diff
@@ -35,7 +32,7 @@
 	<country name="Бахрейн" code="BH"/>
 	<country name="Бангладеш" code="BD"/>
 	<country name="Барбадос" code="BB"/>
-	<country name="Беларусь" code="BY"/>
+	<country name="Белоруссия" code="BY"/>
```

Comment:
@asmecher, I see.  I've checked translation of other countries with Debian package and, you know, that some of that translations are exact, but difficult to find. For example, UK is "Соединенное королевство" (exact translation of the "United Kingdom"), but in Russia we don't use such name. We usually say "Великобритания" ("Great Britain"), but people didn't match "Соединенное королевство" with it. Another example, US is "Соединённые штаты" (exact translation of the "United States"), but in Russia we use "Соединённые штаты Америки" ("United States Of America"), "США" ("USA"), or "Америка" (America), so "Соединённые штаты" also is the name that people do not find.  Should we use the exact translation in this situations? I don't think so. I think we should use the way that people know that countries.

As for Belarus — it's not a problem, we can use "Беларусь", because people understand it.

Should I add the commit with fixed files or @mtub can do it?